pkgs/by-name/mo/monkeysAudio/package.nix

```diff
@@ -6,12 +6,19 @@
 }:
 
 stdenv.mkDerivation (finalAttrs: {
-  version = "11.86";
+  version = "11.87";
   pname = "monkeys-audio";
 
   src = fetchzip {
-    url = "https://monkeysaudio.com/files/MAC_${builtins.concatStringsSep "" (lib.strings.splitString "." finalAttrs.version)}_SDK.zip";
-    hash = "sha256-YFw8TC38Nmdy37Bzo6p43+Ykt9fmvtAcJmZubsFqJHE=";
+    urls =
+      let
+        fileName = "MAC_${lib.concatStringsSep "" (lib.splitString "." finalAttrs.version)}_SDK.zip";
```

Comment:
It'd be nicer I think to use the `url` here, and use it for the web.archive.org link to.

Reply:
Will include once I open up the follow-up PR